server/src/boot/app.ts

```diff
@@ -164,6 +165,13 @@ export default async (
     // (H5P.adapters.express) to function properly.
     app.use(i18nextHttpMiddleware.handle(i18next));
 
+    app.use(async (req: any, res: any, next: express.NextFunction) => {
```

Comment:
Reading a file from the file system on every request will slow down the system if the disk isn't that fast. I think the current language should be stored in some kind of central cache that can be read from here and also manipulated from the settings saving mechanism